ui/client/components/healthCheck.styl

```diff
@@ -0,0 +1,19 @@
+@require "../stylesheets/_variables.styl"
```

Comment:
There is some problem with formatting? tabs / 4 spaces? Evrywere at FE we use 2 spaces.

Reply:
its not formatting, stylus is indent dependent, tabs are intentional, one level - one char. 2,3,4,5,5,156.... spaces = problems.